src/web/src/components/Posts/Issues.tsx

```diff
@@ -14,6 +14,10 @@ const useStyles = makeStyles((theme: Theme) =>
       lineHeight: 'normal',
       fontSize: '1.2rem',
     },
+    GitHubMobile: {
```

Comment:
You might forget to remove this class too.

Reply:
Resolved!